int32 OnOffControl
float32 reference
float32 GoalmsTime

